sdk/eventhub/azure-eventhub-checkpointstoretable/setup.py

```diff
@@ -0,0 +1,74 @@
+from setuptools import setup, find_packages
+import os
+from io import open
+import re
+
+# example setup.py Feel free to copy the entire "azure-template" folder into a package folder named
+# with "azure-<yourpackagename>". Ensure that the below arguments to setup() are updated to reflect
+# your package.
+
+# this setup.py is set up in a specific way to keep the azure* and azure-mgmt-* namespaces WORKING all the way
+# up from python 2.7. Reference here: https://github.com/Azure/azure-sdk-for-python/wiki/Azure-packaging
+
+PACKAGE_NAME = "azure-eventhub-checkpointstoretable"
+PACKAGE_PPRINT_NAME = "Event Hubs checkpointer implementation with Tables Storage"
+
+# a-b-c => a/b/c
+package_folder_path = PACKAGE_NAME.replace("-", "/")
+# a-b-c => a.b.c
+namespace_name = PACKAGE_NAME.replace("-", ".")
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, "_version.py"), "r") as fd:
```

Comment:
https://github.com/Azure/azure-sdk-for-python/blob/e0e8777a1c95d08a5f8b04a179d641c5a355fc13/sdk/eventhub/azure-eventhub-checkpointstoreblob/setup.py#L20-L26

Copy this